.github/workflows/build.yml

```diff
@@ -40,32 +40,67 @@ jobs:
   sign_jag_windows:
     runs-on: windows-signing
     needs: [build_jag]
+    # if: github.event_name == 'release'
     steps:
       - uses: actions/download-artifact@v2
         with:
           name: jag-build
-      - name: Sign jag on Windows
+      - name: Sign jag
         run: |
           dir
-          signtool sign /debug /n "Toitware ApS" /t http://timestamp.digicert.com/ $PWD/jag_windows.exe
+          mv $PWD/jag_windows.exe $PWD/jag.exe
+          signtool sign /debug /n "Toitware ApS" /t http://timestamp.digicert.com/ $PWD/jag.exe
+          powershell Compress-Archive jag.exe jag.zip
```

Comment:
Should this be jaguar.zip to match the installer name or should we just call the installer jag_installer? I think I prefer the latter.